packages/turf-standard-deviational-ellipse/index.js

```diff
@@ -0,0 +1,133 @@
+import { coordAll, featureEach } from '@turf/meta';
+import { getCoords } from '@turf/invariant';
+import { featureCollection, isObject, isNumber } from '@turf/helpers';
+import centerMean from '@turf/center-mean';
```

Comment:
@muziejus I know I asked to changed the name of this module, but maybe `@turf/mean-center` was the correct name for this module... 🤔 

I have no issues if we rename it back to `mean-center` since it looks kinda awkward calling it `centerMean`

@muziejus Thoughts?

Reply:
I have no problem one way or another. Though one benefit of calling it `centerMean` means that I can always just do this:

`meanCenter = centerMean(foo)`

instead of:

`theMeanCenter = meanCenter(foo)`

Reply:
True, but you can also just call it:

```js
const center = meanCenter(foo)
// or
const meanCtr = meanCenter(foo)
```
If we only had 1 center coordinate, also you could find a clever abbreviated term for center, like we've done for `@turf/helpers` methods.

```js
import { point } from '@turf/helpers'
const pt = point([10, 20])
```